birdie/.eslintrc.cjs

```diff
@@ -8,9 +8,8 @@ module.exports = {
 		'airbnb-base',
 		'airbnb-typescript/base',
 		'plugin:@typescript-eslint/eslint-recommended',
-		'plugin:@typescript-eslint/recommended',
 		'plugin:@typescript-eslint/recommended-requiring-type-checking',
-		'plugin:@typescript-eslint/strict',
+		'plugin:@typescript-eslint/strict-type-checked',
```

Comment:
`strict-type-checked` includes both `strict` and `recommended`, hence the removal.